test/path_observer_test.dart

```diff
@@ -331,20 +333,24 @@ observePathTests() {
       model.a.b.c = 'hello, mom';
 
       expect(lastValue, null);
+      Observable.dirtyCheck();
       return new Future(() {
```

Comment:
especially here - new.Future was there to ensure dirtycheck was run

Is test's zone messing up our zone?